geo/src/modules/tiles/app.module.ts

```diff
@@ -38,6 +42,10 @@ import { TileAppService } from "./app.service";
       minimumDataLevel: 4,
       maximumDataLevel: 16,
     }),
+    TerrainTileModule.forRoot({
```

Comment:
Could you use this module in another module like `TerrainAppModule`. You should make it in `/src/modules/terrain`. Because TileAppModule depends on GPU, so we need some workaround for TileAppModule. To do it easily, I think we should separate terrain module as different app module.